boards/FSM/Inc/Io/Io_Brake.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <stdbool.h>
+
+/**
+ * Check if the pressure from MSP-300-2K5-P-3-N-1 exceeds the brake pressure
+ * threshold. If the pressure exceeds threshold, the brake is said to be
+ * actuated.
+ * @return True if the brake is actuated, false if the brake is not actuated
```

Comment:
Since we moved this to `Io_Brake`, the jdocs should be independent of the sensor because that's an implementation detail
 ```suggestion
/**
 * Check if the brake is actuated
 * @return True if the brake is actuated, false if the brake is not actuated
```